docs/contribute/contributing.md

```diff
@@ -0,0 +1,16 @@
+# Contributing to TresJS
```

Comment:
We should probably remove this h1 here since the contribution markdown already has one @colinscz 
![Screenshot 2025-05-13 at 15 19 21](https://github.com/user-attachments/assets/d1b68012-4928-4411-9cad-cebab27330df)

Reply:
Sure @alvarosabu - I just removed it. Can I assume you favor the approach in this commit: e66709359e659d12b7a2a95946795bcd76ef62ee with the data renderer? (See other open discussion)